### DISCUSSION TITLE
Implementing the DDEX standard

### DISCUSSION THREAD

**Discussion by LouisLouiseKay** [funmusicplace/mirlo#883] [Ideas]
This is key for interoperability with other entities like publishers, music distributors, and also to create a pipeline that would allow us to distribute to Tone.audio

Resources here

https://ddex.net/resources/

It's an involved process, but it's best to work on it early on to future proof the platform.

**Comment by tshiunghan** [comment]
See also https://github.com/funmusicplace/mirlo/discussions/160#discussion-5773282

**Comment by LouisLouiseKay** [comment]
Implementation specs 

https://service.ddex.net/doc/PastStandards/MEAD%201.1%20Text.pdf
